extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/SplitPackageProcessor.java

```diff
@@ -70,6 +84,11 @@ void splitPackageDetection(ApplicationArchivesBuildItem archivesBuildItem,
                 continue;
             }
 
+            // skip user defined packages
+            if (packagesToSkip.contains(packageName)) {
```

Comment:
I wonder if it should be a `startsWith` or if we should support `.*`. Rationale is that there is a good chance you will want to ignore your whole root package.

WDYT?

Reply:
@gsmet hmm I was thinking about this as well but in the end went with exact package name. The reason being that we don't really want user to just silence the warning - if possible, they should fix this issue and only add few exceptions for when you cannot help it (like generated code etc). And for such case, it should not be a whole root package :-)

But I am willing to change it if you convince me otherwise.

Reply:
To mitigate #19030 I would surely welcome `.*` but I hope that index related root cause can be sorted out soon.
I do get the idea of not providing wildcard support.
I'm actually rather neutral on this...